ajax/libs/plotly.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "plotly.js",
+  "filename": "plotly.min.js",
+  "version": "1.12.0",
+  "description": "The open source javascript graphing library that powers plotly",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/plotly/plotly.js.git"
+  },
+  "author": "Plotly, Inc.",
+  "keywords": [
+    "graphing",
+    "plotting",
+    "data",
+    "visualization",
+    "plotly"
+  ],
+  "npmName": "plotly.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "plotly.js",
```

Comment:
how about `plotly-geo-assets.js`
`plotly*` is fine now, you can see the other intern's commit